lib/buyers/field.py

```diff
@@ -23,7 +31,7 @@ def get_prep_value(self, value, prepared=False):
             # to_python() unless you are doing a .update() in which case
             # to_python()  is not called and the value is passed in raw.
             if not isinstance(value, HashedData):
-                value = make_password(value, salt=self.salt)
+                value = self._hash(value, salt=self.salt)
```

Comment:
I see that this is what it was before, I just don't see why the instance method is needed.